addon/components/r-button/component.js

```diff
@@ -8,7 +8,7 @@ export default Component.extend(ButtonMixin, {
   layout,
   layoutName: 'components/r-button',
   tagName: 'button',
-  attributeBindings: ['disabled', 'variant'],
+  attributeBindings: ['disabled', 'variant', 'type'],
```

Comment:
Do we need to go through each time we use a button (which isn't a link button) and specify the type?